packages/react-button/src/components/Button/useButtonState.ts

```diff
@@ -32,5 +33,17 @@ export const useButtonState = (draftState: ButtonState) => {
     }
   }
 
-  draftState.disabled = draftState['aria-disabled'] = draftState.disabled || draftState.loading;
+  draftState['aria-disabled'] = draftState.disabled;
+  draftState.disabled = draftState['aria-disabled'] && !draftState.disabledFocusable;
```

Comment:
I think that we can have `disabled` only for `button` tag. Please correct me if I am wrong

Reply:
What do you mean here?

Reply:
`disabled` attribute is applicable for `button`s, but not for `div`:

```tsx
<>
  <button disabled />
  <div disabled /> // TS2322: Type '{ disabled: true; }' is not assignable to type 'DetailedHTMLProps'...
```

In `buttonBehavior()` we are handling this:

```js
buttonBehavior({ as: 'div', disabled: true }).attributes.root
// ⏩ 
// {role: "button", tabIndex: 0, disabled: undefined, aria-disabled: true}
```

https://github.com/microsoft/fluentui/blob/eca0330d791682031d691cdd8fe6fb0a65c159ab/packages/fluentui/accessibility/src/behaviors/Button/buttonBehavior.ts#L10